### DISCUSSION TITLE
install fail on kernel step security onion 2.3.2

### DISCUSSION THREAD

**Discussion by fw587** [Security-Onion-Solutions/securityonion#1944] [Unsupported Versions] [answered]
[hi guys
when installing 2.3.2 on step 373/426 (installing kernel) I got stock and an unkonow erre has occured.
for installing, choose other linux and ubunto (2 seprated time i've tryed to install) on vmware configuring os.

would you please help?
![WhatsApp Image 2020-11-17 at 09 51 00](https://user-images.githubusercontent.com/58883235/99353905-b32c8700-28ba-11eb-9c01-4b2563b4c1a7.jpeg)

**Comment by weslambert** [comment, marked as answer]
To be clear, have you tried installing just stock Centos 7 or Ubuntu 18.04 on the box?
